website/src/static/_redirects

```diff
@@ -0,0 +1 @@
+/* / 200
```

Comment:
for single page application, all requests will go straight making sure we serve the right page